---
name: Tamara
surname: Sternlieb
position: "PhD"
address: "Ciudad de Buenos Aires, Argentina"
phone: +54911 4069 9574
profilepic: ProfilePic.jpeg
email: "tamara.sternlieb@gmail.com"
twitter: TamSternlieb
linkedin: tamara-sternlieb-93727367
date: "`r format(Sys.time(), '%B %Y')`"
aboutme: "Tamara is a scientist specialized in the study of pathogenic parasites, a bioinformatics and system biology enthusiast, eager to expand the toolbox for biotechnology research."
output: vitae::awesomecv
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
```

```{r}
#Do this before running Knit!
#If not installed, install rorcid and usethis packages

#install.packages("rorcid")
#install.packages("usethis")

#Then execute authentification with ORCID. For this you have to be logged in
#ORCID user

#orcid_auth()
#This will open a new browsere window telling you that authentification was
#succesfull and to come back to R.
#In the console a "Bearer" alphanumeric code will appear. Copy the alphanumeric code and have it close by

#Execute the next part in a new sript window:
#library('usethis')
#usethis::edit_r_environ("project")
#This is to edit the R environment of this project alone, but you can use "user" instead too
#A new window will open in the text editor which will be called .REnviron
#Write ORCID_TOKEN="your_alphanumeric_code" with the quoutes
#Save it, close it and restart R

#Now you are authorized to extract info from your ORCID profile


#Also beware of Latex and the latex packages needed for compiling. See https://yihui.org/tinytex/r/#debugging if you get any errors related to *.tex compiling 
```

# Education

```{r}
library('rorcid')

orcid_data <- do.call("rbind",
  rorcid::orcid_educations("0000-0002-1167-5367")$`0000-0002-1167-5367`$`affiliation-group`$summaries
)
orcid_data %>%
  detailed_entries(
    what = `education-summary.role-title`,
    when = glue::glue("{`education-summary.start-date.year.value`} - {`education-summary.end-date.year.value`}"),
    with = `education-summary.organization.name`,
    where = `education-summary.organization.address.city`
  )
# library(tibble)
# tribble(
#   ~ Degree, ~ Year, ~ Institution, ~ Where,
#   "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
#   "Master of Physics", "1893", "Sorbonne Université", "Paris, France",
#   "Master of Mathematics", "1894", "Sorbonne Université", "Paris, France"
# ) %>% 
#   detailed_entries(Degree, Year, Institution, Where)
```

# Work experience

```{r}
library(tibble)
tribble(
  ~ Experience, ~ Year, ~ Institution, ~ Details,
  "PhD student in the Signaling and Adaptive Mechanisms in Trypanosomatids lab", "2015-2020", "INGEBI-CONICET", "Evaluation of the role of adenosine nucleotides in the stress response pathways of *Trypanosoma cruzi*",
  "Bioinformatic Internship in the Protein Physiology lab of the Biochemistry department", "2012-2013", "Universidad de Buenos Aires", "Readapting a C++ script aimed to find repetitive motifs in protein sequences to Python",
  
) %>% 
  detailed_entries(Details, Year, Experience, Institution)
```
# Languages:
  
  * **Spanish**: mother tongue.
  * **English**: advanced speaking, writing and reading.
  * **Hebrew**: intermediate speaking, writing and reading.

# Skills
These are some of the skills and knowledge I aquired during my training, education and courses

  * **Molecular biology assays**:  
      Western blots; PCR; cloning; cellular transfection in different types of cells; bacterial, yeast, parasite and mammalian cells culture; microscopy and fluorescent microscopy; protein quantification and enzymatic activity assessment.
  * **Bioinformatic tools**:  
      Use of on-line databases containing omics information (Ensembl Browser Workshop 2019); Analysis and processing of fluorescence microscopy images with FIJI; Biostatistics; coding and data analysis with R.
  * **Specialyzed knowledge**:  
      Crispr/Cas9 genetic modification technique (several courses between 2016 and 2018);  Molecular system biology; Signal transduction pathways.
  * **Other informatic tools**:  
      Office 365 package; GraphPad Prism; LaTeX; Git; Inkscape; Photoshop.
  * **Public speaking**:  
      I  have attended and presented my scientific work in several national and international conferences, both orally and in poster format.
      
# Publications

```{r}
library('rorcid')
library('tidyverse')
out <- works(orcid_id("0000-0002-1167-5367")) 
#cit <- orcid_citations(orcid="0000-0002-1167-5367",put_code = 73100122)
 tibble(
   Journal = out$`journal-title.value`,
   Title = out$title.title.value,
   Year = out$`publication-date.year.value`,
   Doi = out$url.value
 ) %>% 
  detailed_entries(
  Doi,
  Journal,
  Title,
  Year
)
#(res<-orcid_citations(orcid = "0000-0002-1167-5367"))

#library(dplyr)
#scholar::get_publications("EmD_lTEAAAAJ&EmD_lTEAAAAJ&") %>% 
#  filter(cites > 30) %>% 
#  distinct(title, .keep_all = TRUE) %>%
#  transmute(bibtype = "Article", author = as.character(author),
#            title = as.character(title),
#            journaltitle = as.character(journal), year, key = row_number()) %>% 
#  RefManageR::as.BibEntry() %>% 
#  RefManageR::WriteBib("curie.bib")

#bibliography_entries("curie.bib")
```

